### DISCUSSION TITLE
Bindless texture

### DISCUSSION THREAD

**Discussion by junglie85** [gfx-rs/wgpu#3324] [Q&A]
Are there any examples of how to implement bindless textures? I see there is an extension and it’d be super helpful with my 2D renderer.

**Comment by junglie85** [comment]
Ah, this is what a texture array allows! Is there a max number of array elements or is that driver/hardware dependent?

**Reply by CatCode79** [reply]
I think this is the value you want:
https://docs.rs/wgpu/latest/wgpu/struct.Limits.html#structfield.max_texture_array_layers
